experimental/PropertyDDS/packages/property-dds/src/test/rebasing.spec.ts

```diff
@@ -5,6 +5,7 @@
 
 /* eslint-disable @typescript-eslint/no-unused-expressions */
 import * as crypto from "crypto";
+import { strict as assert } from "assert";
```

Comment:
Since we're already using chai's `expect` pattern in the tests themselves, would it be better to just use chai's assert, rather than node's?